portfolio/src/main/webapp/scripts/templatemo-script.js

```diff
@@ -63,7 +63,7 @@ async function fetchJson() {
 
     json.forEach(comment => {
```

Comment:
in case fetchJson() function is called multiple times, there will be duplicate comments.
I think you should clear the content of comment-list first before adding more childs